packages/ember-auto-import/ts/analyzer.ts

```diff
@@ -192,7 +192,7 @@ export default class Analyzer extends Funnel {
       try {
         transformSync(source, options);
       } catch (err) {
-        if (err.name !== 'SyntaxError') {
+        if (err instanceof Error && err.name !== 'SyntaxError') {
```

Comment:
I understand what you're doing here but I don't like the idea of introducing a subtle behaviour change just to work around types here. I would prefer if you worked around the  types like you did later in the PR. Something like `(err as unknown as {name: string})`? but don't quote me on it 😂

Reply:
But what's the change in behaviour here? In case of a `SyntaxError` it will always be an instance of `Error`, right? And if it's not a `SyntaxError`, then the if condition will not be hit either way. If feel this is the cleaner way to do this than to override the types, given that TS is actually right here.

Reply:
@mansona 👆😏